staging/knative/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: knative
-version: 0.5.2
+version: 1.9.3
```

Comment:
Just the chart version is being bumped here -- if this is a fork of an upstream chart somewhere, I would expect that we would need to update the templates/manifests here? Or, is there an upstream helm chart that we can just use directly instead of forking?

Reply:
It does appear [we have forked this one](https://github.com/mesosphere/charts/tree/master/staging/knative/charts). Is anyone aware of the reason? Maybe we can switch to using an upstream chart directly or something.

Reply:
last i checked there is no upstream helm chart - and the "official/recommended" way of installing is just via plain yaml manifests - https://knative.dev/docs/install/  , this "might" be the reason why we have what we have.

Reply:
@takirala Thanks for confirming and helping me out, I just realized that's why you helped me set it up this way.